tests/app_engine/device/app_engine_device_get_samples/test_app_engine_device_get_samples_default.py

```diff
@@ -0,0 +1,290 @@
+    device_id = astarte_env_vars["device_test_1"]
+    astarte_url = astarte_env_vars["astarte_url"]
+    realm = astarte_env_vars["realm"]
+    jwt = astarte_env_vars["jwt"]
+
+    interface_name = "test.astarte-platform.server.individual.parametric.Datastream"
+
+    for path in list_of_params_endpoints:
+        arg_list = [
+            "astartectl",
+            "appengine",
+            "devices",
+            "get-samples",
+            device_id,
+            interface_name,
+            path,
+            "-t",
+            jwt,
+            "-u",
+            astarte_url,
+            "-r",
+            realm,
+        ]
+        sample_data_result = subprocess.run(arg_list, capture_output=True, text=True)
+        sample_data_result.stdout = _remove_table_border(sample_data_result)
+
+        assert sample_data_result.stdout != ""
+
+        sample_data_result.stdout = re.sub(r"\s+", " ", sample_data_result.stdout).strip()
+        sample_data_result.stdout = sample_data_result.stdout.replace(" ", ",").split(",")
+        sample_data_result.stdout = [parse_element(el) for el in sample_data_result.stdout]
+
+        parsed_elements_list = []
+        for element in sample_data_result.stdout:
+            if element.startswith("["):
+                parsed_elements_list.append(element.replace("[", ""))
+            elif element.endswith("]"):
+                parsed_elements_list.append(element.replace("]", ""))
+
+        list_length = len(sample_data_result.stdout) - 1
+
+        if parsed_elements_list != []:
+            assert parsed_elements_list == map_of_params_data[path]
+        else:
+            assert sample_data_result.stdout[list_length] == map_of_params_data[path]
+
+
+def test_get_sample_data_for_server_individual_nonparametric_datastream(astarte_env_vars):
+    device_id = astarte_env_vars["device_test_1"]
+    astarte_url = astarte_env_vars["astarte_url"]
+    realm = astarte_env_vars["realm"]
+    jwt = astarte_env_vars["jwt"]
+
+    interface_name = "test.astarte-platform.server.individual.nonparametric.Datastream"
+
+    for path in list_of_nonparams_endpoints:
+        arg_list = [
+            "astartectl",
+            "appengine",
+            "devices",
+            "get-samples",
+            device_id,
+            interface_name,
+            path,
+            "-t",
+            jwt,
+            "-u",
+            astarte_url,
+            "-r",
+            realm,
+        ]
+        sample_data_result = subprocess.run(arg_list, capture_output=True, text=True)
+        sample_data_result.stdout = _remove_table_border(sample_data_result)
+
+        assert sample_data_result.stdout != ""
+
+        sample_data_result.stdout = re.sub(r"\s+", " ", sample_data_result.stdout).strip()
+        sample_data_result.stdout = sample_data_result.stdout.replace(" ", ",").split(",")
+        sample_data_result.stdout = [parse_element(el) for el in sample_data_result.stdout]
```

Comment:
These three lines repeat through multiple tests. It would be wise to put them in a function.